### ISSUE TITLE
freeradius - builded with old libssl

### ISSUE THREAD

**Issue by armdn** [felixonmars/archriscv-packages#3728]
Tried to launch newly installed freeradius with radiusd -X command and caught this error:

```
ArchVF2# radiusd -X
libssl version mismatch.  built: 30100040 linked: 30200010
```

Freeradius info:
```
Name            : freeradius
Version         : 3.2.3-5
Description     : The premier open source RADIUS server
Architecture    : riscv64
URL             : https://freeradius.org/
Licenses        : GPL
Groups          : None
Provides        : None
Depends On      : krb5  net-snmp  postgresql-libs  mariadb-libs  talloc  libpcap  libxcrypt  libcrypt.so=2-64  smbclient
Optional Deps   : curl: for REST [installed]
                  freetds: for Sybase and MS SQL
                  hiredis: for redis support [installed]
                  json-c: rlm_rest module [installed]
                  libmemcached: for memcached
                  perl: for Perl [installed]
                  python: for Python [installed]
                  sqlite: for sqlite [installed]
                  unixodbc: for ODBC
                  yubico-c-client: for yubicloud
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 9.66 MiB
Packager        : Felix Yan <felixonmars@archlinux.org>
Build Date      : Mon Dec 18 16:21:55 2023
Install Date    : Sat Apr 13 14:03:59 2024
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
```

System:
`Linux ArchVF2 5.15.2-cwt-5.11.3-2 #1 SMP PREEMPT Fri Mar 22 19:02:42 +07 2024 riscv64 GNU/Linux`

**Comment by felixonmars** [comment]
Fixed in freeradius 3.2.3-8. Thx for reporting!

**Comment by armdn** [comment]
Again this error appears.

`libssl version mismatch.  built: 30200010 linked: 30300000`
